apps/prairielearn/src/lib/db-types.ts

```diff
@@ -536,7 +545,12 @@ export const EnrollmentSchema = z.object({
   course_instance_id: IdSchema,
   created_at: DateFromISOString.nullable(),
   id: IdSchema,
-  user_id: IdSchema,
+  lti_synced: z.boolean(),
+  pending_uid: z.string().nullable(),
+  pending_sub: z.string().nullable(),
+  pending_lti13_instance_id: IdSchema.nullable(),
```

Comment:
Nit, alphabetization of properties.

Reply:
Done in 5880955200e413aa52612c37549a240f9e9d25e8. And a rule in #12622